File: CSharp/modelo/ContaCorrente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QualquerBank.modelo
{
    public class ContaCorrente : Conta
    {
        private const double TaxaDeOperacao = 0.10;

        public ContaCorrente(int numero, int agencia) : base(numero, agencia) { }

        public override bool Sacar(double valor)
        {
           return base.Sacar(valor + TaxaDeOperacao);
        }

        public override void ExibeConta()
        {
            Console.WriteLine("Conta Corrente");
            Console.WriteLine("Numero: " + Numero);
            Console.WriteLine("Agencia: " + Agencia);
            Console.WriteLine("Saldo: " + Saldo);
            Console.WriteLine("Taxa: " + TaxaDeOperacao);
        }
    }
}

File: Java/contas/ContaCorrente.java
package contas;

public class ContaCorrente extends Conta {
	private final double TAXA_DE_OPERACAO = 0.10;
	
	public ContaCorrente(int numero, int agencia) {
		super(numero, agencia);
	}

	@Override
	public boolean sacar(double valor) {
		return super.sacar(valor + TAXA_DE_OPERACAO);
	}

	@Override
	public void exibeConta() {
		System.out.println("Conta Corrente");
		System.out.println("Numero: " + getNumero());
		System.out.println("Agencia: " + getAgencia());
		System.out.println("Saldo: " + getSaldo());
		System.out.println("Taxa: " + TAXA_DE_OPERACAO);		
	}	
}
